File: Anul II/Medii de proiectare si programare/Laborator/lab02/C#/Ubb.BikeContest.Model/src/Team.cs
namespace Ubb.BikeContest.Model;

public class Team : Identifiable<long>
{
    private string _name;

    public Team(long id, string name)
    {
        Id = id;
        _name = name;
    }
    
    public string Name
    {
        get => _name;
        set => _name = value;
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab03/Java/src/main/java/com/ubb/model/Team.java
package com.ubb.model;

public class Team extends Identifiable<Long> {
    private final String name;

    {
        setID(0L);
    }

    public Team(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    @Override
    public String toString() {
        return "Team '" + getName() + "'";
    }
}
